lychee-bin/tests/cli.rs

```diff
@@ -1895,6 +1895,13 @@ mod cli {
             .stderr(contains(
                 "https://github.com/lycheeverse/lychee#non-existent-anchor",
             ))
+            .stderr(contains("fixtures/fragments/sub_dir#non-existing-fragment-1"))
+            .stderr(contains("fixtures/fragments/sub_dir#non-existing-fragment-2"))
+            .stderr(contains("fixtures/fragments/sub_dir_non_existing_1"))
+            .stderr(contains("fixtures/fragments/sub_dir_non_existing_2"))
+            .stderr(contains("fixtures/fragments/empty_dir"))
+            .stderr(contains("fixtures/fragments/empty_dir#non-existing-fragment-3"))
+            .stderr(contains("fixtures/fragments/empty_dir#non-existing-fragment-4"))
```

Comment:
Basically, this is what I would additionally expect here:
```rs
            .stderr(contains("fixtures/fragments/sub_dir"))
            .stderr(contains("fixtures/fragments/sub_dir#a-link-inside-index-html-inside-sub-dir"))
```
Both should succeed, but only one of them seems to be recognized in summary below.

Reply:
I missed that the `empty_dir/` link existed before, so +8 URIs makes sense. And `empty_dir/` succeeded before the PR, now it expectedly fails.

So the two new `sub_dir` URLs could be added to the test, as suggested above, but the result and overall numbers make all sense.